src/index.ts

```diff
@@ -52,33 +51,57 @@ app.message(
     // user.displayName = context.profile.display_name;
     // user.slackID = context.channel.creator;
 
-    const user = {
+    const findUser = {
       realName: context.profile.real_name,
       displayName: context.profile.display_name,
       slackID: context.channel.creator,
     };
-    const query = {
-      slackID: user.slackID,
+    const insertedUser = await mongoEntityMgr.findOneAndReplace(
+      "users",
+      { slackID: findUser.slackID },
+      findUser,
+      { upsert: true }
+    );
+
+    const findMessage = {
+      ts: message.ts,
+      content: message.text,
+      userId: insertedUser.value._id,
+      channelId: message.channel,
     };
-    await mongoEntityMgr
-      .findOneAndReplace("users", query, user, { upsert: true })
-      .then((res) => {
-        console.log({ res });
-      })
-      .catch((err) => {
-        console.log({ err });
-      });
-
-    await app.client.chat
+    const insertedMessage = await mongoEntityMgr.findOneAndReplace(
+      "users_posts",
+      { ts: findMessage.ts },
+      findMessage,
+      { upsert: true }
+    );
+
+    const postTLRes = await app.client.chat
       .postMessage(msgOption)
-      .then((res) => {
-        if (res.ok) console.log("msg: ok ✅");
-      })
       .catch((err) => {
         console.error({ err });
         console.log(err.data.response_metadata);
+        return { ok: false, ts: null, channel: null, message: null };
       });
 
+    if (postTLRes.ok) {
+      console.log("msg: ok ✅");
+
+      const findTL = {
+        ts: postTLRes.ts,
+        bindedChannelID: postTLRes.channel,
+        contents: postTLRes.message.blocks,
+        usersPostID: insertedMessage.lastErrorObject.upserted,
+      };
+      const insertedTLRes = await mongoEntityMgr.findOneAndReplace(
```

Comment:
```suggestion
      const resInsertedTL = await mongoEntityMgr.findOneAndReplace(
```
個人的にはこうしたいしん TL とかの後に R が入ると若干見辛いしん

Reply:
了解っす、暫定的に書きまくってたら自分でも混乱してました🤯

Reply:
:done: